recipes-kernel/linux/kernel-devsrc.bbappend

```diff
@@ -0,0 +1,31 @@
+do_install_prepend() {
+    cd ${S}
+    if [ "${@bb.utils.vercmp_string_op(d.getVar("KERNEL_VERSION"), '5.10', '>=')}" = "True" ]; then
+        touch arch/arm64/kernel/vdso/gettimeofday.S
+        touch arch/arm64/kernel/module.lds
+    fi
+}
+
+do_install_append() {
+    if [ "${@bb.utils.vercmp_string_op(d.getVar("KERNEL_VERSION"), '5.10', '>=')}" = "True" ]; then
+        cd ${S}
+        if [ "${ARCH}" = "arm64" ]; then
+            install -D arch/arm64/kernel/vdso/*gettimeofday.* \
+                $kerneldir/build/arch/arm64/kernel/vdso/
+        fi
+        install -D lib/vdso/gettimeofday.* $kerneldir/build/lib/vdso/
+        install -D K* $kerneldir/build/
```

Comment:
Please comment that you intend Kbuild and Kconfig (or explicitly specify these names).